tests/test_parse_enex.py

```
@@ -0,0 +1,535 @@
+        '<div>memo only</div>'
+        '</en-note>'
+    )
+    enex = _wrap_export(_make_minimal_note("見出し無し", body=body))
+    path = _write_enex(tmp_path, enex)
+    note = next(parse_enex(path))
+    md = note_to_markdown(note)
+    ai_start = md.index("## 🤖 Evernote AI 構造化情報")
+    transcription_start = md.index("## 🗣️ 生の文字起こし")
+    ai = md[ai_start:transcription_start].strip()
+    body_only = ai.removeprefix("## 🤖 Evernote AI 構造化情報").strip()
+    assert body_only in {"", "（構造化情報なし）"}
```

Ruff (0.15.12)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)